xbmc/settings/AdvancedSettings.h

```diff
@@ -380,6 +380,8 @@ class CAdvancedSettings : public ISettingCallback, public ISettingsHandler
     False to show at the bottom of video (default) */
     bool m_videoAssFixedWorks;
 
+    bool m_openGlDebugging;
```

Comment:
I know this is "how it was always done" but is there any actual reason for this to be a public data member as opposed to just having a getter?

Reply:
I've left this for now. I agree that getters should be used though.

Reply:
You've got to start somewhere ;-) (that's as far as I'm going to push it)